test/contracts/pricer/get_price_point.js

```diff
@@ -41,8 +41,8 @@ module.exports.perform = (accounts) => {
     eurPriceOracle = contracts.eurPriceOracle;     
     await pricer.setPriceOracle(pricer_utils.currencies.usd, usdPriceOracle.address, { from: opsAddress });
     await pricer.setPriceOracle(pricer_utils.currencies.eur, eurPriceOracle.address, { from: opsAddress });
-    await usdPriceOracle.setPrice(usdPrice, { from: opsAddress });
-    await eurPriceOracle.setPrice(eurPrice, { from: opsAddress });
+    await usdPriceOracle.setPrice(usdPrice);
```

Comment:
can we have the test without any reference to an actual existing fiat?

Reply:
👍